entity-management-web/src/test/java/eu/europeana/entitymanagement/validation/EntityFieldsCleanerTest.java

```diff
@@ -0,0 +1,57 @@
+package eu.europeana.entitymanagement.validation;
+
+import static eu.europeana.entitymanagement.testutils.BaseMvcTestUtils.loadFile;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.IOException;
+
+import javax.annotation.Resource;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import eu.europeana.api.commons.error.EuropeanaApiException;
+import eu.europeana.entitymanagement.common.config.AppConfigConstants;
+import eu.europeana.entitymanagement.common.config.LanguageCodes;
+import eu.europeana.entitymanagement.config.AppConfig;
+import eu.europeana.entitymanagement.definitions.model.Agent;
+import eu.europeana.entitymanagement.exception.EntityCreationException;
+import eu.europeana.entitymanagement.normalization.EntityFieldsCleaner;
+import eu.europeana.entitymanagement.testutils.BaseMvcTestUtils;
+import eu.europeana.entitymanagement.web.service.MetisDereferenceService;
+
+@SpringBootTest
```

Comment:
Specify component classes relevant for this test, instead of loading the entire application context.

Reply:
for this test it is not easy to declare the minimum number of required beans

Reply:
@gsergiu See #61